validator/internal/attester_service_mock.go

```diff
@@ -6,11 +6,12 @@ package internal
 
 import (
 	context "context"
+	reflect "reflect"
```

Comment:
it always likes reflect to be down there for some reason...

Reply:
yeah not sure why, I stopped trying to understand all its linting decisions lol